pom.xml

```diff
@@ -0,0 +1,768 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.capitalone.dashboard</groupId>
+  <artifactId>hygieia-iac-terraform-collector</artifactId>
+  <name>${project.groupId}:${project.artifactId}</name>
+  <description>Terraform Cloud App Collector</description>
+  <url>https://hygieia.github.io/Hygieia/getting_started.html</url>
+  <packaging>jar</packaging>
+  <version>1.0.0-SNAPSHOT</version>
+
+  <parent>
+    <groupId>com.capitalone.dashboard</groupId>
+    <artifactId>Hygieia</artifactId>
+    <version>3.0.1</version>
+  </parent>
+
+  <organization>
+    <name>Hygieia</name>
+    <url>https://www.github.com/Hygieia/</url>
+  </organization>
+
+  <scm>
+    <connection>scm:git:https://github.com/Hygieia/hygieia-scm-github-collector.git</connection>
```

Comment:
please fix the url to match the collector

Reply:
Done